src/frontend/components/SøknadsSteg/OmDeg/OmDeg.tsx

```diff
@@ -63,74 +62,58 @@ const OmDeg: React.FC = () => {
                 )}
             </KomponentGruppe>
 
-            {skjema.felter.telefonnummer.erSynlig && (
-                <KomponentGruppe dynamisk>
-                    <span id={skjema.felter.telefonnummer.id}>
-                        <SkjemaFeltInput
-                            felt={skjema.felter.telefonnummer}
-                            visFeilmeldinger={skjema.visFeilmeldinger}
-                            labelSpråkTekstId={omDegSpørsmålSpråkId.telefonnummer}
-                            bredde="M"
-                            type="tel"
-                            maxLength={20}
-                        />
-                    </span>
-                    <JaNeiSpm
-                        skjema={skjema}
-                        felt={skjema.felter.oppholderSegINorge}
-                        spørsmålTekstId={omDegSpørsmålSpråkId[OmDegSpørsmålId.oppholderSegINorge]}
-                    />
+            <KomponentGruppe dynamisk>
```

Comment:
Her burde vi legge på en sjekk om det neste JaNeiSpm er synlig 😊 dette for at aria-live skal tre inn når komponenten rendres. Før sørget telefonnummer.erSynlig dette for begge disse spørsmålene.

Reply:
Da er dette lagt på 😊